Formula/boost-bcp.rb

```diff
@@ -2,8 +2,8 @@ class BoostBcp < Formula
   desc "Utility for extracting subsets of the Boost library"
   homepage "https://www.boost.org/doc/tools/bcp/"
   # Please add to synced_versions_formulae.json once version synced with boost
-  url "https://boostorg.jfrog.io/artifactory/main/release/1.77.0/source/boost_1_77_0.tar.bz2"
-  sha256 "fc9f85fc030e233142908241af7a846e60630aa7388de9a5fafb1f3a26840854"
+  url "https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.bz2"
```

Comment:
Why not sync all the boost formulae upgrades?

Reply:
Because that pull request would never end, I think it's much more likely to be be mergable like this.

Reply:
Got you, good point.